src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -668,6 +688,73 @@ InferenceEngine::Precision FullyConnected::getRuntimePrecision() const {
     return getMaxPrecision(inputPrecisions);
 }
 
+bool FullyConnected::canUseSparseWeights() {
+    if (useSparseWeights == false) {
+        return useSparseWeights;
+    }
+
+    if (!impl::cpu::x64::mayiuse(impl::cpu::x64::avx512_core_amx))
+        return false;
+
+    if (isDynamicNode()) {
+        return false;
+    }
+
+    auto inputDataType = DnnlExtensionUtils::IEPrecisionToDataType(getOriginalInputPrecisionAtPort(DATA_ID));
+    auto weightsDataType = DnnlExtensionUtils::IEPrecisionToDataType(getOriginalInputPrecisionAtPort(WEIGHTS_ID));
+    if (!one_of(inputDataType , memory::data_type::u8, memory::data_type::s8) || weightsDataType != memory::data_type::s8) {
```

Comment:
Please use ov::Precision. Not necessary to convert to DNNL representation.

Reply:
changed, thanks